app/channels/tims_notifications_channel.rb

```diff
@@ -0,0 +1,21 @@
+class TimsNotificationsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "tims_notifications_channel"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.